lib/executor/helpers.go

```diff
@@ -81,10 +80,10 @@ func validateStages(stages []Stage) []error {
 //
 // TODO: emit the end-of-test iteration metrics here (https://github.com/loadimpact/k6/issues/1250)
 func getIterationRunner(
-	executionState *lib.ExecutionState, logger *logrus.Entry, _ chan<- stats.SampleContainer,
-) func(context.Context, lib.VU) {
-	return func(ctx context.Context, vu lib.VU) {
-		err := vu.RunOnce(ctx)
+	executionState *lib.ExecutionState, logger *logrus.Entry,
+) func(context.Context, lib.ActiveVU) {
+	return func(ctx context.Context, vu lib.ActiveVU) {
```

Comment:
There's an TODO on line 103 here that should be removed in this PR

Reply:
You mean plain delete it since the error is now fixed, or are there other `context cancelled` errors that should be investigated? (Also, why comment on this line instead of 103?)

Reply:
yeah, plain delete it, it referenced the error you're fixing with this PR

Reply:
Resolved in fbf60b81.